src/transformers/naive_split.cc

```diff
@@ -53,21 +58,13 @@ namespace detail {
 	}
 
 	naive_split_transformer::naive_split_transformer(size_t num_chunks, size_t num_workers) : num_chunks(num_chunks), num_workers(num_workers) {
+		assert(num_workers > 0);
 		assert(num_chunks >= num_workers);
```

Comment:
This precondition is no longer necessary, maybe replace it by `num_chunks > 0`?

Reply:
Done!